data/Dockerfiles/letsencrypt/root/srv/docker.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This should just be placed in `data/Dockerfiles/letsencrypt/docker.sh`. 
With the `COPY `command you can decide where it is copied to.